_docs/howto/configure/gsuite-group-collection.md

```diff
@@ -19,16 +19,23 @@ service account just for this functionality:
     1.  On the **Create service account** dialog that appears, set up your service account:
         1.  Enter a service account name.
         1.  Select the **Enable G Suite Domain-wide Delegation** checkbox.
+        1.  If you haven't already configured your project's OAuth consent screen, enter a product name
+            to display on the consent screen, then click **Create**. To change the product name or add
+            details to the consent screen later, edit your
+            [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent) settings.
+            
+            <img src="../../images/howto/create-service-account.png" alt="create service account window with product name field highlighted">
         1.  Click **Create**.
     1.  To create and download a JSON key for the service account:
         1.  Click **More** on the service account row, then click **Create key**.
-        <img src="../../images/howto/create-key.png" alt="more menu with create key highlighted">
+            <img src="../../images/howto/create-key.png" alt="more menu with create key highlighted">
```

Comment:
Actually the better way to do this (in markdown and not HTML) is to use this [syntax](https://guides.github.com/features/mastering-markdown/), e.g.

`[![Documentation Categories Screenshot](images/readme/docs_categories_screenshot.jpg)](#)`

Please fix throughout.